python/setup.py

```diff
@@ -301,12 +301,11 @@ def get_option_from_sys_argv(option_str):
       classifiers=[
           "Programming Language :: Python",
           "Programming Language :: Python :: 3",
```

Comment:
I think some setup.py changes got lost again here. The good news is that after this PR is merged, the problem should be solved going forward.

Reply:
I fixed this.